api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -582,83 +571,4 @@ private void validateEventOwnerSelectorUnchanged(final EventType original, final
             throw new InvalidEventTypeException("event_owner_selector can't be set back to null");
         }
     }
-
-    private void validateSchema(final EventTypeBase eventType) throws InvalidEventTypeException {
```

Comment:
I moved all this code into `SchemaService` since it's now used by both `SchemaController` and.`EventTypeController`. It's still possible to update the schema using `PUT /event-types/{name}`.